project_issue_code/__init__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Michael Viriyananda
+#    Copyright 2016 OpenSynergy Indonesia
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def create_code_equal_to_id(cr):
+    """
```

Comment:
Please put these hooks into a file like the conventions: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#installation-hooks